themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties

```
@@ -0,0 +1,47 @@
+passwordResetBodyHtml=<p> Někdo právě požádal o změnu pověření vašeho účtu {2}. Pokud jste to vy, klikněte na odkaz níže, abyste je resetovali.</p><p><a href="{0}">Odkaz na obnovení pověření </a></p><p> Platnost tohoto odkazu vyprší během {1} minut.</p><p> Pokud nechcete obnovit vaše pověření, ignorujte tuto zprávu a nic se nezmění.</p>
+executeActionsSubject=Aktualizujte svůj účet
+executeActionsBody=Váš administrátor vás požádal o provedení následujících akcí u účtu {2}: {3}. Začněte kliknutím na níže uvedený odkaz.\n\n{0}\n\nPlatnost tohoto odkazu je {4}.\n\nPokud si nejste jisti, zda je tento požadavek v pořádku, ignorujte tuto zprávu.
+executeActionsBodyHtml=<p>Váš administrátor vás požádal o provedení následujících akcí u účtu {2}: {3}. Začněte kliknutím na níže uvedený odkaz.</p><p><a href="{0}">Odkaz na aktualizaci účtu.</a></p><p>Platnost tohoto odkazu je {4}.</p><p>Pokud si nejste jisti, zda je tento požadavek v pořádku, ignorujte tuto zprávu.</p>
+eventLoginErrorSubject=Chyba přihlášení
+eventLoginErrorBody=Někdo se neúspěšně pokusil přihlásit k účtu {0} z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventLoginErrorBodyHtml=<p>Někdo se neúspěšně pokusil přihlásit k účtu {0} z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.</p>
+eventRemoveTotpSubject=Odebrat TOTP
+eventRemoveTotpBody=V účtu {0} bylo odebráno nastavení OTP z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventRemoveTotpBodyHtml=<p>V účtu {0} bylo odebráno nastavení OTP z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.</p>
+eventUpdatePasswordSubject=Aktualizace hesla
+eventUpdatePasswordBody=V účtu {0} bylo změněno heslo z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventUpdatePasswordBodyHtml=<p>V účtu {0} bylo změněno heslo z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.</p>
+eventUpdateTotpSubject=Aktualizace OTP
+eventUpdateTotpBody=V účtu {0} bylo změněno nastavení OTP z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventUpdateTotpBodyHtml=<p>V účtu {0} bylo změněno nastavení OTP z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.</p>
+
+requiredAction.CONFIGURE_TOTP=Konfigurace OTP
+requiredAction.terms_and_conditions=Smluvní podmínky
+requiredAction.UPDATE_PASSWORD=Aktualizace hesla
+requiredAction.UPDATE_PROFILE=Aktualizace profilu
+requiredAction.VERIFY_EMAIL=Ověření e-mailu
+
+# units for link expiration timeout formatting
+linkExpirationFormatter.timePeriodUnit.seconds=sekundy
+linkExpirationFormatter.timePeriodUnit.seconds.1=sekunda
+linkExpirationFormatter.timePeriodUnit.minutes=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.1=minuta
+linkExpirationFormatter.timePeriodUnit.minutes.2=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.3=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.4=minuty
+linkExpirationFormatter.timePeriodUnit.hours=hodiny
+linkExpirationFormatter.timePeriodUnit.hours.1=hodina
+linkExpirationFormatter.timePeriodUnit.days=dny
```

```suggestion
linkExpirationFormatter.timePeriodUnit.days=dní
```
and alternatives for 2, 3, and 4 (similarly to what you have for _minutes_ above)